Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -2134,6 +2296,35 @@ public void UnRegister(string sessionName, ProductUserId userIdToUnRegister)
             sessionInterface.UnregisterPlayers(ref unregisterOptions, sessionName, OnUnregisterCompleteCallback);
         }
 
+        /// <summary>
+        /// Unregisters a user from a Session.
+        /// This will then inform the user to leave the room with P2P messages.
+        /// An unregistered user will no longer be part of the Session with EOS Game Services.
+        /// Uses <paramref name="sessionName"/> as the ClientData to identify the local Session name to unregister the user from.
+        /// When called by the Owner of a Session, the user is unregistered from the Session in EOS Game Services.
+        /// Other users should call this to keep members of the Session current.
+        /// P2P messages can be used to inform the Owner and users when to run this method.
+        /// </summary>
+        /// <param name="sessionName">Local Session name to unregister the user from.</param>
+        /// <param name="userIdsToUnRegister">
+        /// The ProductUserId of the users to unregister.
+        /// If this array is empty, the function exits without performing an operation.
+        /// </param>
+        public void UnRegister(string sessionName, ProductUserId[] userIdsToUnRegister)
+        {
+            if (userIdsToUnRegister.Length == 0)
+            {
+                return;
+            }
+
+            UnregisterPlayersOptions unregisterOptions = new UnregisterPlayersOptions();
+            unregisterOptions.SessionName = sessionName;
+            unregisterOptions.PlayersToUnregister = userIdsToUnRegister;
+
+            SessionsInterface sessionInterface = EOSManager.Instance.GetEOSPlatformInterface().GetSessionsInterface();
+            sessionInterface.UnregisterPlayers(ref unregisterOptions, sessionName, OnUnregisterCompleteCallback);
+        }
```

Comment:
I would change the signature of this function to emphasize that it is the user id(s) who are being unregistered from the session. That may be the only kind of unregistration that takes place, but the clarity seems like a win.

Reply:
Sure! The Sessions Interface calls it `UnregisterPlayers`, so let's add "Players" and "Player" to the end of these function names.

Reply:
Also it should be "Unregister" not "UnRegister".

Reply:
That's an easy change, sure. Maintained it from its previous name.